docs/packaging/submitting-a-pull-request.mdx

```diff
@@ -37,25 +42,47 @@ git log -1
 
 ## Creating the Pull Request
 
-In the package folder, run `git push`.
-
-:::note
-
-If you've created your own branch, as recommended, the cli tool will show you a new command to create and push to a remote branch matching the local one. Run this.
-
-:::
-
-Once the commit is successfully pushed, you'll notice that a URL will be provided that will immediately allow you to create a pull request with your changes.
-
-Run `git status` one last time to make sure your branch is clean. If it is:
-
-1. Open the link.
-2. Fill in a summary of your changes (usually the same as the commit message).
-3. Link any relevant issues:
-   - If you want to link this pull request to an existing issue, simply mention it in the PR summary: `The inclusion of <somepackage> fixes #123`.
-   - If you need a change to depend on another change, mention it in the PR summary too: `Depends on #234`.
-4. Double check everything
-5. Create the pull request!
+There are multiple ways to create a Pull Request with GitHub, either from the website, or from the command line.
+
+<Tabs groupId="opening-prs">
+  <TabItem value="website" label="Website">
+    <p>
+    In the package folder, push your local changes to a remote branch:
+
+    <CodeBlock language="bash">
+      git push
+    </CodeBlock>
+    </p>
+    <Admonition type="info">
+      If you've created your own branch, as recommended, the CLI tool will show you a new command to create and push to a remote branch matching the local one. Run this.
+    </Admonition>
+    <p>
+    Once the commit is successfully pushed, you'll notice that a URL will be provided that will immediately allow you to create a pull request with your changes.
+
+    Run `git status` one last time to make sure your branch is clean. If it is:
+
+    1. Open the link.
+    2. Fill in a summary of your changes (usually the same as the commit message).
+    3. Link any relevant issues:
+       - If you want to link this pull request to an existing issue, simply mention it in the PR summary: `The inclusion of <somepackage> fixes #123`.
+       - If you need a change to depend on another change, mention it in the PR summary too: `Depends on #234`.
+    4. Double check everything
+    5. Create the pull request!
+    </p>
+
+  </TabItem>
+  <TabItem value="github-cli" label="GitHub CLI">
+    <p>
+    In the package folder, run the `gh` command:
+    </p>
+    <CodeBlock language="bash">
+      gh pr create
+    </CodeBlock>
+    <p>
+    This will start an interactive session to create your pull request step-by-step.
+    </p>
```

Comment:
Something I also remembered that initially confused me:

If `gh` ever asks you to choose between your fork or the `getsolus/packages` repository as the [default remote repository](https://cli.github.com/manual/gh_repo_set-default), you should choose `getsolus/packages`. This doesn't change any `git remote` settings; it merely tells `gh` which repository to query and send your PRs to.

Reply:
That only applies to members of the Solus Staff team with direct push access. Other users should use their fork.

Reply:
Could just add one line in italics below the info box